packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -38,8 +39,9 @@ export type TransferPanelMainInputProps =
   }
 
 export function TransferPanelMainInput(props: TransferPanelMainInputProps) {
-  const { errorMessage, maxButtonProps, value, ...rest } = props
+  const { errorMessage, maxButtonProps, value } = props
```

Comment:
Since wew are now only requiring these 3 props from `TransferPanelMainInputProps`, I think it would be ideal to sanitise `TransferPanelMainInputProps` to only have these keys and not extend it from `React.InputHTMLAttributes<HTMLInputElement>`

Reply:
Good point